hj350-strassen.md

```diff
@@ -0,0 +1 @@
+Nice town, now living in Mamer
```

Comment:
Why did you leave? @hj350

Reply:
I grown up in Strassen and since 13 years now in Mamer @Treverer